android/src/com/mapswithme/util/UiUtils.java

```diff
@@ -367,6 +368,9 @@ public static int getStatusBarHeight(@NonNull Context context)
   {
     int result = 0;
     Resources res = context.getResources();
+    // TODO: find a better way of getting system status bar height
+    // https://github.com/organicmaps/organicmaps/issues/3611
+    @SuppressLint({"DiscouragedApi", "InternalInsetResource"})
```

Comment:
@arnaudvergnet

Reply:
This will be removed in https://github.com/organicmaps/organicmaps/pull/3495 in favor of system insets.

Reply:
Thanks, @arnaudvergnet.